tests/swagger_generation/test_marshmallow_to_swagger.py

```diff
@@ -8,6 +8,7 @@
     :license: MIT, see LICENSE for details.
 """
 import enum
+import unittest
```

Comment:
see `__main__` statement where we call unittest

Reply:
We should just remove the `__main__` statement